src/index.js

```diff
@@ -187,6 +187,8 @@ class Node extends EventEmitter {
           return each(this.modules.discovery, (d, cb) => d.start(cb), cb)
         }
         cb()
+
+        this.emit('online')
```

Comment:
I'm not convinced that `online`/`offline` is the right wording here. In js-ipfs we have:

![image](https://user-images.githubusercontent.com/1211152/27760667-15acefa8-5e45-11e7-8c57-c935a5a47356.png)

start/stop sounds better because `online` has a lot of assumptions today. To me, start/stop say "hey, the thing you set up, is running". 

Thoughts?